### PR TITLE
Make repo subrepoable

### PR DESCRIPTION
**Pull request by amirebrahimi**
I moved the files to the root and updated the README with instructions on how to clone directly into a project. This is what we did for [EditorVR](https://github.com/Unity-Technologies/EditorVR/) and [AutoLOD](https://github.com/Unity-Technologies/AutoLOD). I'm looking to make use of the graph visualizer for another project, but will need to subrepo it, so I figured I'd offer this back.

### CODE REVIEW

**Comment by romainf-unity** [comment]
Hmm it's a bit of a problem since I moved them into a specific directory a few months ago: 5aa9fa8fbc261c2278bfbdfac90b19dc3481d6d9

I'll have to check with the Package team as well, I know I want to make this a package, but I don't know don't know what should be the directory pattern for that.

**Comment by romainf-unity** [comment]
Also, instead of moving all the files, it's possible to do a sparse checkout:
https://stackoverflow.com/questions/600079/how-do-i-clone-a-subdirectory-only-of-a-git-repository#13738951

**Comment by amirebrahimi** [comment]
Unfortunately sparse checkouts wouldn't be maintained for subrepos. The problem isn't with what directories get checked out, but instead from the need to check out at the root, which precludes a larger Unity project from being version controlled.

If you need to have a separate folder structure for packages, then a good solution is to have a separate repo that subrepos this one (or vice versa) and imports it at a specific location (i.e. if graph-visualizer is subrepo, then graph-visualizer-project would import it at Assets/GraphVisualizer). You'd be back to having what you have now while still allowing other projects to subrepo.

File: README.md [hunk 1]
```diff
@@ -4,8 +4,13 @@ The PlayableGraph Visualizer window can be used to display any *PlayableGraph*.
...
 ## Setup ##
-- Download the release that matches your current Unity version, or the latest if there your Unity version is more recent than the latest release.
-- Copy the content of this repos Asset folder into a the Asset folder of an Unity Project.  You will need to do this for every project.
+- Download the release that matches your current Unity version, or the latest if your Unity version is more recent than the latest release.
+- Copy the content of this repo into a folder called GraphVisualizer in your Unity Project.  You will need to do this for every project.
+
+OR
+
+- From the command line change directory to your project's `Assets` directory.
+- Run `git clone https://github.com/Unity-Technologies/graph-visualizer` **(Note HTTPS)**
```
Comment (romainf-unity) [inline README.md]: I don't think it's needed now to precise "https", it's fairly common now, and users will copy/paste the line anyway.

**Review by romainf-unity — CHANGES_REQUESTED** [review]
Minor change to fix in the README, but I'll approve this PR (it is in line with the structure imposed by the package manager)

**Comment by amirebrahimi** [comment]
Should be good now

**Review by romainf-unity — APPROVED** [review]

**Pull request merged**